action.yml

```diff
@@ -17,6 +17,14 @@ inputs:
       this should be the tip of the pull request, obtainable by running
       `git rev-parse ${GITHUB_SHA}^2`"
     required: true
+  mode:
```

Comment:
do we even need regex mode?

Reply:
not anymore, but I'm keeping it around for now so that PyTorch CI doesn't break while we wait for https://github.com/pytorch/pytorch/pull/55569 to land; we can definitely remove it later